keylime/ca_util.py

```diff
@@ -151,6 +152,25 @@ def cmd_mkcert(workingdir: str, name: str, password: Optional[str] = None) -> No
         os.chdir(cwd)
 
 
+def cmd_import_priv(workingdir: str, priv_pem_file: str, lastserial: int) -> None:
+    cwd = os.getcwd()
+    try:
+        with open(priv_pem_file, "rb") as priv_pem:
+            pem = priv_pem.read()
+        priv = read_private(False)
+        private_key = load_pem_private_key(pem, None, default_backend())
+        priv[0]["ca"] = private_key.private_bytes(
+            encoding=serialization.Encoding.PEM,
+            format=serialization.PrivateFormat.PKCS8,
+            encryption_algorithm=serialization.NoEncryption(),
+        )
+        priv[0]["lastserial"] = int(lastserial) + 1
+        fs_util.ch_dir(workingdir)
```

Comment:
For consistency reasons I would put the `ch_dir` before the `read_private()` because all other functions have it also in this order to read the key from the `workingdir`. Maybe there's also a benefit of not loosing possibly already revoked keys once importing/changing a private key?

Reply:
Ok. Fixed. That's it?

Reply:
LGTM now.